crossdock/client/ctxpropagation/behavior.go

```diff
@@ -281,14 +272,18 @@ func (h *multiHopHandler) Handle(reqMeta *json.ReqMeta, body interface{}) (inter
 	return map[string]interface{}{}, resMeta, err
 }
 
-func buildRPC(t crossdock.T, ch *tchannel.Channel) (rpc yarpc.RPC, tconfig server.TransportConfig) {
+func buildRPC(t crossdock.T) (rpc yarpc.RPC, tconfig server.TransportConfig) {
 	fatals := crossdock.Fatals(t)
 
 	self := t.Param("ctxclient")
 	subject := t.Param("ctxserver")
 	fatals.NotEmpty(self, "ctxclient is required")
 	fatals.NotEmpty(subject, "ctxserver is required")
 
+	// always need a TChannel
```

Comment:
This comment probably isn't needed, buildRPC is called everytime, so `ch` will get created every time.